src/map/pc.h

```diff
@@ -744,6 +744,7 @@ END_ZEROED_BLOCK;
 #define pc_isinvisible(sd)    ( (sd)->sc.option&OPTION_INVISIBLE )
 #define pc_is50overweight(sd) ( (sd)->weight*100 >= (sd)->max_weight*battle->bc->natural_heal_weight_rate )
 #define pc_is90overweight(sd) ( (sd)->weight*10 >= (sd)->max_weight*9 )
+#define pc_maxhp_cap(sd) ( status->get_maxhp_cap_entry(pc->class2idx(sd->status.class), sd->status.base_level)->value )
```

Comment:
```
#define pc_maxhp_cap(sd) ( status->get_maxhp_cap_entry(pc->class2idx((sd)->status.class), (sd)->status.base_level)->value )
```

Please add parentheses around the macro argument to be safe against unusual expressions

Reply:
oops. done. thank you!